pages/settings/SettingsLog.qml

```diff
@@ -164,7 +178,11 @@ Rectangle {
                     wrapMode: TextEdit.Wrap
                     readOnly: true
                     function logCommand(msg){
-                        msg = log_color(msg, "lime");
```

Comment:
Ternary operator would be more readable here IMO and more compact, but that's just personal style.

```
msg = log_color(msg, MoneroComponents.Style.blackTheme ? "lime" : "green")
consoleArea.append(msg);
```

or even in one line but that's pushing readability :D

```
consoleArea.append(log_color(msg, MoneroComponents.Style.blackTheme ? "lime" : "green"));
```